cordformation/src/main/kotlin/net/corda/plugins/External.kt

```diff
@@ -0,0 +1,28 @@
+package net.corda.plugins;
+
+import org.gradle.api.tasks.Input
+
+open class External {
+
+    @get:Input
+    var containerName: String = "external-service"
+
+    @get:Input
+    var servicePorts: List<Int> = listOf(8080)
+
+    @get:Input
+    var containerImage = ""
```

Comment:
I think this one should be  an `@Optional` `Property<String>` without a convention.